examples/darwin-framework-tool/templates/tests/ciTests.json

```diff
@@ -38,6 +38,14 @@
         "Disabled because the power source configuration cluster is now deprecated and not present in all-clusters",
         "Test_TC_PSCFG_1_1",
         "Test_TC_PSCFG_2_1",
-        "Test_TC_PSCFG_2_2"
+        "Test_TC_PSCFG_2_2",
+        "Disabled due to SmokeCOAlarm not being enabled in Matter.framework for now:",
```

Comment:
I don't think you need these anymore following #28105

Reply:
There are other bugs that exist at the moment, so keep them for now.